types/tress/tress-tests.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="node" />
 
-import { tress } from "tress";
+import * as tress from "tress";
```

Comment:
Can you change this to `import tress = require("tress");`?

Reply:
@DanielRosenwasser, why is it necessary? Currently, it uses Typescript/ES6 module import style. See below for another example of the same syntax.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0e02e7b65b259990ed3e9b14a16f053a6ce5a1fa/types/needle/needle-tests.ts#L1